ledger/participant-integration-api/src/main/scala/platform/indexer/ExecuteUpdate.scala

```diff
@@ -372,7 +373,8 @@ class PipelinedExecuteUpdate(
 
   private[indexer] val flow: ExecuteUpdateFlow =
     Flow[OffsetUpdate]
-      .mapAsync(1)(prepareUpdate)
+      .mapAsync(2)(prepareUpdate)
```

Comment:
I think the parallelism of this stage should be configurable.

Reply:
exposed in `com.daml.platform.indexer.IndexerConfig`